packages/scxa-tsne-widget/src/TSnePlotWidget.js

```diff
@@ -142,7 +169,7 @@ TSnePlotWidget.propTypes = {
 
 
 TSnePlotWidget.defaultProps = {
-  atlasUrl: `https://www.ebi.ac.uk/gxa/sc/`,
+  atlasUrl: `http://localhost:8080/gxa/sc/`,
```

Comment:
Hi @lingyun1010, this will move back to the previous URL(https://www.ebi.ac.uk/gxa/sc/), right?